src/targets/Transpiler.JIT.ts

```diff
@@ -39,4 +40,9 @@ export class LuaTranspilerJIT extends LuaTranspiler52 {
     public transpileDestructingAssignmentValue(node: ts.Expression): string {
         return `unpack(${this.transpileExpression(node)})`;
     }
+
+     /** @override */
+    public transpileSpreadElement(node: ts.SpreadElement): string {
+        return LuaTranspiler.prototype.transpileSpreadElement(node);
```

Comment:
I would prefer you just copy paste the entire function from the transpiler (only one line anyway) here for clarity. I understand the idea behind prototype but I'm not sure that is the best way to do this.